README.md

```diff
@@ -117,7 +116,7 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The second specifies that you will be using postGRES instead of MySQL.
 ```
```

Comment:
It also specifies that the Rails app will be created inside of the current directory (i.e., `rails new` will not create a brand new `tunr` directory).